### DISCUSSION TITLE
_expansions

### DISCUSSION THREAD

**Discussion by repay** [postalserver/postal#2027] [Development discussions]
Hi! Is _expansions on api not working or am I doing something wrong?

http://apiv1.postalserver.io/structures/message.html

I've tried 
host/api/v1/messages/message?_expansions=plain_body
host/api/v1/messages/message?_expansions=true

**Comment by willpower232** [comment]
I think you need to send the expansions in the request body as follows
![image](https://user-images.githubusercontent.com/1619102/174492930-bf08ac89-07eb-438a-aa43-67d9d1e33f54.png)
